Add one-liner info about components of KubeFlow to docs (#4055)
kubeflow/website: content/en/docs/components/pipelines/_index.md

+++
title = "Kubeflow Pipelines"

> description = "Building and Deploying Scalable ML Workflows in Docker Containers"

Reviewer: ```suggestion
description = "Platform for building and deploying ML workflows using containers"
```

Reviewer 2: We should not call Kubeflow Pipeline a platform, since "Kubeflow is a collection of tools", and KFP is just one tool from Kubeflow Foundation.

Reviewer 3: We also use OCI instead of docker.
